plasmapy/tests/test_requirements.py

```diff
@@ -1,140 +0,0 @@
-"""Tests for the consistency of requirements."""
```

Comment:
Oh yeah, I should probably have mentioned this - I got rid of it to get the openastro workflow to finish. That one failed because the requirements files don't get included as part of the wheel/sdist. So the `open` we do at the test prep level there completely breaks the test collection for the whole package! It'd work if we moved that to a fixture with a `skipif` on locating the file, but then the parametrize logic has to be changed and I just wanted to keep the momentum going here.